components/gfx/display_list/mod.rs

```diff
@@ -331,6 +330,27 @@ impl StackingContext {
             scrolls_overflow_area: scrolls_overflow_area,
             scroll_policy: scroll_policy,
             layer_id: layer_id,
+        };
+        StackingContextLayerizer::layerize(&mut stacking_context);
```

Comment:
I think I'd prefer a name more along the lines of `make_layer` rather than "layerize"--"layerize" is a nice abbreviation for technical discussion, but those reading the code may not know what it means.